### DISCUSSION TITLE
Hello all!

### DISCUSSION THREAD

**Discussion by mantasbandonis** [OpenMined/courses#112] [Introduce Yourself]
Good evening/morning wherever you are! :) 

My name is Mantas, and I am a graduate student in Artificial Intelligence and Digital Healthcare from Austria. At my university, we do not really cover topics about differential privacy or something similar in any way (more about the ethics of AI). Therefore I am thrilled to find a course about this with such an astonishing quality. I could definitely feel the passionate intro to this topic, which got me going. It is also possible to rely on so many examples and topics that were brought up so far, and it made it very clear and easy to understand the concepts.

I was sitting until 3 AM now to finish the second lesson! Going to optimise my sleep now, so that Netflix does not take any time away from it, as it is said in the lesson ;) 

Really great course so far, thank you!

**Comment by yemikifouly** [comment]
Hello Mantas! 😃 Woot woot!!! Comments like these get us going and make us feel that all the work that went into developing the course was worth it! I am so glad you are enjoying the course and yes. Here at OpenMined we are definitely all very passionate about these topics, starting, of course, with our leader Andrew Trask! 🙂 

Really, I am psyched that you are enjoying the course and do post in the discussion board if you have any question or any feedback for us. 🙂 Welcome to OpenMined!

**Comment by em-blue** [comment]
Thank you so much @mantasbandonis ! :-) That means a lot - I hope you enjoy the rest of the course, looking forward to hearing your thoughts
